src/Google.Api.Gax/Api/Gax/BackoffSettings.cs

```diff
@@ -0,0 +1,116 @@
+        /// <remarks>
+        /// The default value of this property is <see cref="TimeSpan.Zero"/>.
+        /// </remarks>
+        /// <exception cref="ArgumentOutOfRangeException">The value is negative.</exception>
+        public TimeSpan Delay
+        {
+            get { return _delay; }
+            set
+            {
+                if (value < TimeSpan.Zero)
+                {
+                    throw new ArgumentOutOfRangeException(nameof(value), value, "Delay must not be negative");
+                }
+                _delay = value;
+            }
+        }
+
+        /// <summary>
+        /// The multiplier to apply to the delay on each iteration; must be greater than or equal to 1.0.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// The default value of this property is 1.0.
+        /// </para>
+        /// <para>
+        /// As an example, a multiplier of 2.0 with a delay of 0.1s on an RPC which fails three times before
+        /// succeeding would lead to an initial delay between the first response and the second request
+        /// of 0.1s; a delay between the second response and the third request of 0.2s, and a delay
+        /// between the third response and the fourth request.
+        /// </para>
+        /// </remarks>
+        /// <exception cref="ArgumentOutOfRangeException">The value is not a real number, or is less than 1.</exception>
+        public double DelayMultiplier
+        {
+            get { return _delayMultiplier; }
+            set
+            {
+                if (value < 1 || double.IsInfinity(value) || double.IsNaN(value))
+                {
+                    throw new ArgumentOutOfRangeException(nameof(value), value, "Delay multiplier must be a real number greater than or equal to 1");
+                }
+                _delayMultiplier = value;
+            }
+        }
+
+        /// <summary>
+        /// The maximum delay to use. If the increasing delay due to the delay multiplier exceeds this,
+        /// this maximum is used instead.
+        /// </summary>
+        /// <remarks>
+        /// The default value of this property is <see cref="TimeSpan.Zero"/>.
+        /// </remarks>
+        /// <value>The maximum delay. Must not be negative.</value>
+        /// <exception cref="ArgumentOutOfRangeException">The value is negative.</exception>
+        public TimeSpan MaxDelay
+        {
+            get { return _maxDelay; }
+            set
+            {
+                if (value < TimeSpan.Zero)
+                {
+                    throw new ArgumentOutOfRangeException(nameof(value), value, "Maximum delay must not be negative");
+                }
+                _maxDelay = value;
+            }
+        }
+
+        /// <summary>
+        /// Creates a clone of this instance.
+        /// </summary>
+        /// <returns>A clone of this instance.</returns>
+        public BackoffSettings Clone() =>
+            new BackoffSettings
```

Comment:
My evolved convention (which you can probably find exceptions for :) is

"If it all fits on one line, do so. Otherwise, break after =>"

I don't think there's anything like a widespread convention for this yet.